### ISSUE TITLE
Feature request: add option for devices without compass

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sky-map-team/stardroid#19]
```
I really love SkyMap. Thanks to the developers for developing such a nice app.

This is no bug but a feature request. At the moment SkyMap can be used on 
devices without compass only in manual mode. In auto mode changing the back 
azimuth is not possible.

Still the acceleration sensor should provide enough information to align the 
screen to the pointing direction, even without compass. It just has to be 
aligned once which could be done in the manual mode. It would be nice if there 
was an option to enable such a feature. This option could also be automatically 
enabled if no compass is detected.

If you think this is not feasible feel free to close the request.
```

Original issue reported on code.google.com by `lehc...@gmail.com` on 23 Oct 2013 at 1:31

**Comment by jaydeetay** [comment]
So in this case the map would move, but only up and down?  That's probably do-able.  Will give it some thought when we switch over to using the gyro.

**Comment by ErnieRael** [comment]
(I started writing email; then noticed github, started issue; then noticed would be duplicate)

I unwittingly got an android phone without a compass. (long story...)

It seems like the phone does altitude OK, but without a compass it, of course, can't do azimuth.

The feature request is: have a way that only compass direction (azimuth) is manual.

For example, one idea is to have separate auto/manual buttons for altitude and azimuth. If both are manual, it would behave like the current manual mode.

BTW, currently when switching from manual to automatic, the compass direction is changed back to ???. If on the switch it remembered the manual setting of compass, then it would actually work as desired (I think). In that case, no UI work would be needed.

**Comment by jaydeetay** [comment]
Thanks Ernie for the feature request.  I think it might be possible to do this.  I'm  a little doubtful about how useful it is but I guess you wouldn't ask for it if you didn't think you'd use it.

**Comment by ErnieRael** [comment]
I've only just started with _sky map_ (forced to get a smart phone, since regional tracfone will stop supporting 2G), I've been using _Stellarium_ on a PC for a while. Admittedly, being a new user, I'm guessing at the usefulness. From what I've experienced so far, I'll use such a feature a lot.

But I am guessing, thus the idea of doing it without any UI changes to reduce effort and allow some usage experimentation. I'd be happy to alpha/beta test. (But note: "...At this time, the testing program has enough testers...")

**Comment by jaydeetay** [comment]
Stellarium is a fantastic bit of software.  I was briefly working for the
same organization as its author (10 years ago when I had delusions of being
an astronomer).

Thanks for telling me about the testing program being full up - I wasn't
aware of that and will fix it.  We set a cap just to have some measure of
control but it's really low (500 users I think) and if people are willing
to try the beta we're certainly happy to have them test it!

**Comment by jaydeetay** [comment]
Right.  I've just increased the number of testers, so once that change
percolates through please do try again.  Sorry about the inconvenience.

**Comment by SuicSoft** [comment]
Maybe use this camera for this ?

**Comment by jaydeetay** [comment]
Can you explain a little more Suici how you'd do this?

**Comment by SuicSoft** [comment]
@jaydeetay Don't really know but it is how Samsung phones without compass can take panoramas.

**Comment by jaydeetay** [comment]
I see.  They have the advantage of being able to stitch the images.  That
could be hard to do in real time... Especially with pictures taken at
night.  I don't think this is feasible unfortunately.

**Comment by jaydeetay** [comment]
I still think this is feasible, but it could also be very confusing.  Going to defer this to 2.0 when we can do a better job of providing an 'warm welcome' to guide the user.

**Comment by jaydeetay** [comment]
Actually I don't know why I wrote that last comment...I'm not sure it's actually do-able.
